### PR TITLE
J-245: fix bag create podcast without genre

### PR DESCRIPTION
**Pull request by yaroslav207**

### CODE REVIEW

File: frontend/src/store/configurate-podcast/actions.ts [hunk 1]
```diff
@@ -22,7 +22,7 @@ const create = createAsyncThunk<void, PodcastFormPayload, AsyncThunkConfig>(
     const podcast = await podcastApi.create({
       userId: (<User>auth.user).id,
       description: podcastPayload.description,
-      genreId: Number(podcastPayload.genre),
+      genreId: (podcastPayload.genre === null) ? null : Number(podcastPayload.genre),
```
Comment (what1s1ove) [inline frontend/src/store/configurate-podcast/actions.ts:25]:
```suggestion
      genreId: podcastPayload.genre ?? Number(podcastPayload.genre),
```

**Review by what1s1ove — CHANGES_REQUESTED** [review]

File: frontend/src/store/configurate-podcast/actions.ts (hunk 1 above)
Comment (yaroslav207) [reply frontend/src/store/configurate-podcast/actions.ts:25]: > 

This way we cannot to send null to the server.

Comment (yaroslav207) [reply frontend/src/store/configurate-podcast/actions.ts:25]: genreId: number | null

Comment (what1s1ove) [reply frontend/src/store/configurate-podcast/actions.ts:25]: got you

**Review by what1s1ove — APPROVED** [review]

**Pull request merged**